lib/misc/api/api_token.dart

```diff
@@ -42,24 +45,25 @@ Future<String?> getToken() async {
   return Future.value(value);
 }
 
+/// Get refresh token from storage
 Future<String?> getRefreshToken() async {
   String? value = await storage.read(key: _refreshToken);
   return Future.value(value);
 }
 
+/// Get wheter to auto login from storage
 Future<bool> getAutoLogin() async {
   bool value = await storage.read(key: _autoLogin) == 'true';
   return value;
 }
 
+/// Get wheter user is admin from storage
```

Comment:
Minor typo, check all for similar typo wheter -> whether

Reply:
oopsies